sycl/gdb/libsycl.so-gdb.py

```diff
@@ -51,35 +51,38 @@ def value(self, arg):
 class HostAccessor(Accessor):
```

Comment:
Regarding the changes in this (sycl/gdb/libsycl.so-gdb.py) file:
The modifications are related to Xmethods. The pretty-printers are defined in the same file, but they are conceptually different. Just to avoid confusion, I’d suggest saying “xmethods” instead of “pretty-printers” in the commit message.